### PR TITLE
chore: small refactor works

### PR DESCRIPTION
**Pull request by tien**

### CODE REVIEW

**Comment by alanrsoares** [comment]
ok, now I get why those chainIds were put as hex

File: packages/stargate/src/signingStargateClient.ts [hunk 1]
```diff
@@ -240,9 +240,7 @@ export class SifSigningStargateClient extends SigningStargateClient {
     senderAddress: string,
     recipientAddress: string,
     transferAmount: Coin,
-    // default values from old sdk
-    // 0x1 is mainnet 0x3 is ropsten
-    ethChainId: number | string = 0x1,
+    ethChainId = 1,
```
[the remark this answers is not in the document]
Comment (tien) [reply packages/stargate/src/signingStargateClient.ts:243]: opps merged, will create a change for this now

[the remark this answers is not in the document]
Comment (alanrsoares) [reply packages/stargate/src/signingStargateClient.ts:243]: `MAINNET` is actually called `HOMESTEAD` for ethereum

[the remark this answers is not in the document]
Comment (alanrsoares) [reply packages/stargate/src/signingStargateClient.ts:243]: it shoud reference the network specifc name as different chains will have other names to reference mainnets and testnets

**Pull request merged**